ddtrace/monkey.py

```diff
@@ -60,8 +61,8 @@
     'pylons': False,
     'pyramid': False,
 
-    # Standard library modules off by default
-    'logging': False,
+    # Auto-ennable logging if they configure `DD_LOGS_INJECTION=true`
+    'logging': config.logs_injection,
```

Comment:
Suggestion: making a separate PR that would change add `DD_LOGS_INJECTION` will make reviews simpler.